### ISSUE TITLE
Concerned about clone

### ISSUE THREAD

**Issue by StupidSaurusRex** [xd4rker/MinerBlock#46]
So I always try multiple extensions to see what I find best suits my needs... I came upon this extension: https://chrome.google.com/webstore/detail/miner-blocker-block-coin/ejpcojkcallnhphinmknkaoojohidegf

and I found it to be IDENTICAL to minerBlock and it got me concerned that there was some code 'theft', particularly given their lack of any 'About' information... As well as the fact that their version 1.1.3 just happened to come out days after the version 1.1.3 was released for this extension... Hoping someone can sort this out.

**Comment by xd4rker** [comment]
MinerBlock is [open source](https://github.com/xd4rker/MinerBlock/blob/master/LICENSE.md), anyone is allowed to make their own clone out of it, only with the condition of preserving the license notice, which these guys seem to have violated... I will try to contact the developer, and if that fails, I will report it to Google...

Thank you :)

**Comment by StupidSaurusRex** [comment]
Oh, interesting! I did not know that. I just noticed yours was identical,
and has been updated and included about and contact information. So it just
annoyed me in seeing possible appropriation of someone's code and possibly
including malware (something that seems increasingly common in extensions).

Thank you very much for informing me of the open source nature of the code,
teaches me to be more thorough in learning about things before jumping to
conclusions!

Take care and be well!

Thank you kindly,

~Jacob
